index.bs

```diff
@@ -1875,8 +1875,14 @@ To <dfn>check if cookie-based debugging is allowed</dfn> given a
 
 <h3 algorithm id="obtaining-context-origin">Obtaining context origin</h3>
 
-To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|, return |node|'s [=node navigable=]'s
-[=navigable/top-level traversable=]'s [=navigable/active document=]'s [=origin=].
+To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|:
+
+1. Optionally, return the embedded origin if it exists.
```

Comment:
It'd be helpful for people reading the spec to have some definition of `embedded origin` here, either in a note or in a link.